src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -423,11 +423,11 @@ An GoogleComputeEngine NodeSource can now be added using the *Create Node Source
 WARNING: When ProActive server is stopped, GoogleComputeEngine instances will be automatically deleted. And when ProActive server is restarted, the infrastructure will be recovered as per the previous settings. The required GoogleComputeEngine instances will be re-created and re-configured.
 
 
-===== Load Sharing Facility (LSF) infrastructure
+==== Load Sharing Facility (LSF) infrastructure
```

Comment:
We should agree whether "infrastructure" is capitalized or not in the section titles, we use both lower and upper case so far.